interface-spec.md

```diff
@@ -60,10 +60,10 @@ Abstract interface.
 - `string .termType` contains the constant `"Literal"`.
 - `string .value` the text value, unescaped, without language or type (example: `Brad Pitt`)
 - `string .language` the language as lowercase [BCP47](http://tools.ietf.org/html/bcp47) string (examples: `en`, `en-gb`) or an empty string if the literal has no language.
-- `IRI .datatype` the datatype of the literal
+- `string .datatype` the IRI of the literal
```

Comment:
I am somewhat late to this party, but thought I would clarify my intent with the original (which was way back before IRI was a term of art, and we used "RDF URI Reference" as a stand-in). I was aware that for the definition of RDF graph to be mathematically tight, it was necessary for the three sets:  RDF URI References (IRIs); blank nodes; and literals to be disjoint by construction. The definition of blank nodes https://www.w3.org/TR/2004/REC-rdf-concepts-20040210/#section-blank-nodes ensures that they are disjoint. RDF URI references (IRIs) are defined as strings https://www.w3.org/TR/2004/REC-rdf-concepts-20040210/#section-Graph-URIref, whereas literals have named components (which strings do not). https://www.w3.org/TR/2004/REC-rdf-concepts-20040210/#section-Graph-Literal.
Thus all three are disjoint, and I believed at the time that this was sufficiently rigorous and not sloppy. This key design is carried forward into the RDF 1.1 definitions, and the reader who believes them to be sloppy is, IMO, not reading carefully.